src/backend/distributed/utils/citus_stat_tenants.c

```diff
@@ -225,6 +229,7 @@ AttributeTask(char *tenantId, int colocationId, CmdType commandType)
 	strncpy_s(AttributeToTenant, MAX_TENANT_ATTRIBUTE_LENGTH, tenantId,
 			  MAX_TENANT_ATTRIBUTE_LENGTH - 1);
 	AttributeToCommandType = commandType;
+	QueryStartClock = clock();
```

Comment:
previously I made a case for moving AttributeQueryIfAnnotated to the end of distributed_planner, though as a result we are excluding query planning CPU time because the clock starts after planning.

The good news is that this is consistent between local and remote execution, which might be better than only counting it in one of the cases, but mainly wanted to point it out.

Reply:
I think it's nice we're consistent.
If later it turns out to be necessary, we can add a small function call that just records the time.